x/profiles/types/models/dtag_transfer_request.go

```diff
@@ -9,31 +9,31 @@ import (
 
 // DTagTransferRequest represent a dtag transfer request between two users
 type DTagTransferRequest struct {
-	DTagToTrade   string         `json:"dtag_to_trade" yaml:"dtag_to_trade"`
-	CurrentOwner  sdk.AccAddress `json:"current_owner" yaml:"current_owner"`
-	ReceivingUser sdk.AccAddress `json:"receiving_user" yaml:"receiving_user"`
+	DTagToTrade string         `json:"dtag_to_trade" yaml:"dtag_to_trade"`
+	Receiver    sdk.AccAddress `json:"receiver" yaml:"receiver"`
+	Sender      sdk.AccAddress `json:"sender" yaml:"sender"`
 }
 
-func NewDTagTransferRequest(dtagToTrade string, currentOwner, receivingUser sdk.AccAddress) DTagTransferRequest {
+func NewDTagTransferRequest(dtagToTrade string, receiver, sender sdk.AccAddress) DTagTransferRequest {
 	return DTagTransferRequest{
-		DTagToTrade:   dtagToTrade,
-		CurrentOwner:  currentOwner,
-		ReceivingUser: receivingUser,
+		DTagToTrade: dtagToTrade,
+		Receiver:    receiver,
+		Sender:      sender,
 	}
 }
 
 // Equals returns true if the two requests are equals. False otherwise
 func (dtagTR DTagTransferRequest) Equals(other DTagTransferRequest) bool {
```

Comment:
Can we rename the `dtagTR` parameter to `request` instead? I think it would make the code easier to read